hooks/useFormatDate.ts

```diff
@@ -0,0 +1,20 @@
+import { useFormatter } from 'next-intl';
```

Comment:
Created this new hook to use the `useFormatter` from `next-intl` to format dates